docs/content/theme/pfe-theme-slots.md

```diff
@@ -97,66 +93,66 @@ At first glace, there seem to be lots of "gotchas" related to web components, an
 
     Using the pfe-card template example above, let's explore what would happen if we called the slots out of order:
 
-    ```
-    // my-web-page.html template
+    my-web-page.html template:
+
+    ```html
     <pfe-card>
-        <div slot="pfe-card--footer">World</div>
-        <div slot="pfe-card--header">Hello</div>
+        <p slot="pfe-card--footer">World</p>
+        <h3 slot="pfe-card--header">Hello</h3>
     </pfe-card>
     ```
 
-
     will render the content like this:
 
-    <div style="background:#eee;padding:30px;width: 250px;">
-        <div>Hello</div>
-        <div>World</div>
-    </div>
-
-
+    <pfe-card>
+      <p slot="pfe-card--footer">World</p>
+      <h3 slot="pfe-card--header">Hello</h3>
+    </pfe-card>
 
 ## Styling Slots
 
-The lines blur between shadow DOM & light DOM when slots are involved. Basically if you add the attribute `slot=" "` to a regular HTML element inside a web component, you are opening a window to allow styles from the web component to style that thing. It only applies directly to the item with the slot name on it though, nothing nested inside it.
+If you add a `slot` to a light DOM element, you are opening a window to allow styles from outside the web component to style that section.  That means the typical encapsulation of a web component flies out the window. You can influence the styles using the `::slotted` selector but it only applies to the direct children of the slot.
```

Comment:
I'm confused who the "you" is in this sentence. Is it the developer building the component or the dev building the page with a component? Given the documentation page, I'm guessing its a dev building a component.

Can you "add slots to light DOM elements"? Or do you add slots to the shadow DOM, which allows content to be placed in the light DOM that then appears in the template of the component?

```suggestion
While building a component, if a `slot` is added, it opens a window to allow styles from outside of the component to style that section.  Slotted content foregoes the benefits of encapsulated styles. The component stylesheet may influence the styles using the `::slotted` selector, but it only applies to the direct children of the slot.
```